source/ncengine/graphics2/diligent/pass/PassTypes.h

```diff
@@ -28,11 +28,12 @@ struct ShaderPaths
 
 enum class PassType : uint8_t
 {
-    None,
-    Material,
-    Wireframe,
-    PostProcess,
-    UI
+    None            = 0,
```

Comment:
Made these flags so we can combine misc passes for filtering in the future, if needed. Wireframe registration was specifying != Material and != PostProccess. Easier in my mind to add an inclusion when we add a new misc pass, rather than add an exclusion if we add something else (like SkinnedMaterial here).